README.md

```diff
@@ -25,6 +27,57 @@ A test of `mkdocs` for hosting the OpenFlight docs
   mike set-default latest
   ```
 
+### Container
+- Prerequisite
+  `docker and docker-compose`
+
+- Start the application
+  ```bash
+  # To run in foregorund
+  cd docs.openflighthpc.org
+  docker-compose up -d 
```

Comment:
This looks the same as the "run in background" command, is this as expected?

Reply:
thanks, Updated